src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyGraphOperation.cs

```diff
@@ -155,6 +188,58 @@ private IEnumerable<DependencyDetail> FilterToolsetDependencies(IEnumerable<Depe
             return dependencies;
         }
 
+        private void LogBasicNodeDetails(DependencyGraphNode node, string indent)
```

Comment:
>private void [](start = 8, length = 12)

Nit: If it is not too big it would be nice to add a sample of how would this look like.